app/api/schema.py

```diff
@@ -26,6 +30,20 @@ class Query(graphene.AbstractType):
     vendor = graphene.relay.Node.Field(VendorNode)
     vendors = DjangoFilterConnectionField(VendorNode)
 
+    servings = graphene.List(
+        ServingNode,
+        timetable=graphene.String(),
+        vendor=graphene.String(),
+        date=graphene.String()
+    )
+
+    def resolve_servings(self, args, context, info):
+        timetable = Timetable.objects.get(slug=args['timetable'])
+        vendor = Vendor.objects.get(slug=args['vendor'])
+        date = datetime.strptime(args['date'], '%Y-%m-%d').date()
+
+        return ServingAutoUpdate.get_servings(timetable, vendor, date)
+
```

Comment:
@kola-er I was wondering if this logic could be moved out of this file, to probably where it's crud is at. Same as `servings = graphene.List .....`